clamav-mirror/lighttpdhost.conf

```diff
@@ -7,3 +7,7 @@ mimetype.assign = (
 )
 
 index-file.names = ( "index.htm" )
+
+server.modules += ( "mod_accesslog" )
+
+accesslog.filename  = "/dev/stderr"
```

Comment:
lighthttpd doesn't output to stdout for whatever reason so stderr has to be used
https://redmine.lighttpd.net/issues/2731